src/components/hooks/use-popup.tsx

```diff
@@ -0,0 +1,14 @@
+import { useContext } from 'react'
+import { PopupContext } from 'store/popup-context'
+
+const usePopup = () => {
```

Comment:
These hooks can be created in the same file that the context was created, So we can omit exporting the context.